apps/i18n/studio/en_us.json

```diff
@@ -565,7 +565,6 @@
   "setBackgroundCornered": "set corners background",
   "setBackgroundDots": "set dots background",
   "setBackgroundGraffiti": "set graffiti background",
-  "setBackgroundSpace": "set space background",
```

Comment:
Same question - we don't use this string anywhere?

Reply:
Answered above.

Reply:
Ohhhh.  I see now.  I didn't realize they were exactly the same and since the file is not perfectly alphabetized and collapsed, I couldn't see the remaining strings that we want to keep.